### ISSUE TITLE
Add Worldedit

### ISSUE THREAD

**Issue by Connor-Colenso** [GTNewHorizons/GT-New-Horizons-Modpack#10574]
### Your GTNH Discord Username

Colen

### Your Pack Version

2.1.2.4-14-pre

### Mod Name

Worldedit

### The Mod to Add

This is an incredibly handy tool that I don't think people appreciate until they can use it well. You can replace blocks in a self defined area, generate abstract 3D shapes, copy entire GT multis, delete large amounts of blocks etc. Having this tool for users in creative I feel would be a godsend as I find myself adding it to nearly every instance for personal testing purposes.

A case I've used it a lot recently is switching out the DTPF coils in dev for instance. Or building a MEBF faster because I can do //walls to instantly place the required coils in a hollow wall and the mufflers on top.

**Future versions of minecraft have things like /fill built into the game for a reason, this is simply a more advanced version of that. Has never caused me any crashes/issues in the year+ I've been using it alongside GTNH.**

![image](https://user-images.githubusercontent.com/52056774/176306548-60f76dcc-f119-4805-a862-83dde3cd84ad.png)

![image](https://user-images.githubusercontent.com/52056774/176306837-28d2af59-6cfd-4f34-a62f-4029e4f83619.png)

Important info:

https://www.curseforge.com/minecraft/mc-mods/worldedit

"Can I include WorldEdit in my modpack?
You are free to include WorldEdit, without having to asking for explicit permission. 
A link back to this page or [our GitHub Sponsors](https://github.com/sponsors/EngineHub) would be appreciated, however.

Can I edit the source code?
Visit our [GitHub repository](https://github.com/enginehub/worldedit). WorldEdit is licensed under **GPLv3** and you can compile it with Gradle."

### Final Checklist

- [X] This mod is released under an open source license. Yes, this is required now.
- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying *I like this mod, please reconsider adding it* will prompt us to investigate and reopen it once we confirm your report.
- [X] I believe there is nothing similar in the pack already, or the existing solution isn't good enough.
- [X] I understand this change request may not attract enough attention and thus not be implemented.
- [X] I understand this change request may be rejected due to other community members thinking it's inappropriate.
- [X] I believe this feature would make the pack better.
- [ ] I'm the official maintainer or original author of this mod for minecraft 1.7.10.

**Comment by chochem** [comment]
I guess the main question you should answer is: what is the effect for the 99% who find this completely useless? Does it add any bloat? (load time etc)

**Comment by boubou19** [comment]
people who need it have it already. Dead weight for many users imo, but no real issue against it. It'll be disabled on the officials tho.

**Comment by Connor-Colenso** [comment]
> I guess the main question you should answer is: what is the effect for the 99% who find this completely useless? Does it add any bloat? (load time etc)

Well it adds no items, just commands. It's a small mod after all. All new mods will add to loading time but can't say I have noticed any change personally. My main reason for this is to keep up with modern MC having these features built in with commands like /fill, they can be really useful.

**Comment by Connor-Colenso** [comment]
> people who need it have it already. Dead weight for many users imo, but no real issue against it. It'll be disabled on the officials tho.

I would say us having worldedit already installed on the officials proves it's value.

**Comment by boubou19** [comment]
it just proves we use original WE plugin, which is half broken to work with TEs. Main reason it's here was to allow for world guard on delta, but mytown made that obsolete. Then the plugin got carried on the other officials because of laziness. It is very rare that one uses WE to do stuff on the officials, except maybe on eta for some trolling.

**Comment by Connor-Colenso** [comment]
> it just proves we use original WE plugin, which is half broken to work with TEs.
I mean if you do stupid shit with it then yea, it will misbehave. Copying very NBT heavy stuff is a no-go but that is just standard for modded MC. I copied 18 fusion reactors with hatches + wires in SP and it all worked fine. Almost all public MC servers throw worldedit in because of it's usefulness. Maybe you don't use it much but I'm staff on officials as well and use it a lot. 

I also uncorrupted a chunk by using worldedit to delete the offending block, saving a rollback/admin intervention which was quite useful.
